backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -162,22 +162,29 @@ def resolve_fetch_failures_0_members(
     if still_draft_cset_ids:
         print(f"Fetch attempted for the following csets, but they still remain drafts and thus still have not had their"
               f" members expanded yet: {', '.join([str(x) for x in still_draft_cset_ids])}")
-    true_failed_cset_ids: Set[int] = set(failed_cset_ids) - still_draft_cset_ids
+    non_draft_failure_ids: Set[int] = set(failed_cset_ids) - still_draft_cset_ids
+
+    # Filter by only if has been finalized longer than we would expect it should take for expansion to be available
+    # noinspection PyUnboundLocalVariable
+    still_draft_csets: List[Dict] = [
+        x['properties'] for x in csets_and_members['OMOPConceptSet']
+        if x['properties']['codesetId'] in non_draft_failure_ids]
+    final_failure_ids: Set[int] = get_csets_over_threshold(still_draft_csets, int(expansion_threshold_seconds / 60))
 
     # Close out
-    if true_failed_cset_ids and loop:
+    if final_failure_ids and loop:
         # todo: maybe this is not optimal. Maybe what we should do is check all the expression items and see if 100% are
         #  cases of isExcluded=true and includeDescendants=false. This should be the only case in which a cset is
         #  finalized and has 0 members.
         print(f"2 hours have passed. No members were fetched for the following concept sets, but given the length "
               f"of time passed, members should have been available by now, and we assume that these concept sets do"
-              f" not actually have members. Reporting resolved: {', '.join([str(x) for x in failed_cset_ids])}")
+              f" not actually have members. Reporting resolved: {', '.join([str(x) for x in final_failure_ids])}")
         comment = 'Success result: No members after 2 hours. Considering resolved.'
-        _report_success(failed_cset_ids, failure_lookup, comment, use_local_db)
-    elif true_failed_cset_ids:
+        _report_success(list(final_failure_ids), failure_lookup, comment, use_local_db)
+    elif final_failure_ids:
         raise RuntimeError('Attempted to resolve fetch failures for the following concept sets, but was not able to do '
                            'so. It may be that the Enclave simply has not expanded their members yet:\n\n'
-                           f'{", ".join([str(x) for x in failed_cset_ids])}')
+                           f'{", ".join([str(x) for x in final_failure_ids])}')
```

Comment:
Bugfix: was printing the wrong IDs when failed. Was including csets that were long-lived drafts.